docs/hub/configuration.md

```diff
@@ -0,0 +1,235 @@
+
+## EamHub Custom Resource Configuration
+{: #cr}
+
+The main configuration for {{site.data.keyword.ieam}} is done through the EamHub custom resource, particularly the **spec** field of that custom resource.
+
+This document assumes:
+* The namespace you are running these commands for is in where the {{site.data.keyword.ieam}} management hub operator is deployed.
+* The EamHub custom resource name is the default **ibm-edge**. If it is different, alter the commands to replace **ibm-edge**.
+* The binary **jq** is installed, which ensures that the output is displayed in a readable format.
+
+
+The defined default **spec** is minimal, only containing the license acceptance, which you can view with:
+```
+$ oc get eamhub ibm-edge -o yaml
+...
+spec:
+  license:
+    accept: true
+...
+```
+
+### Operator Control Loop
+{: #loop}
+
+The {{site.data.keyword.ieam}} Management Hub operator runs in a continuous idempotent loop to synchronize the current state of resources with the expected state of resources.
+
+Due to that continuous loop, you need to understand two things when you try to configure your operator-managed resources:
+* Any change to the custom resource will be read asyncronously by the control loop. After you make the change, it might take a few minutes for that change to be enacted through the operator.
+* Any manual change that is made to a resource the operator controls might be overwritten (undone) by the operator enforcing a specific state. 
+
+Watch the operator pod logs to observe this loop:
+```
+oc logs $(oc get pods | grep ibm-eamhub-operator | awk '{print $1}') --tail 20 -f
+```
+{: codeblock}
+
+When a loop finishes, it generates a **PLAY RECAP** summary. To see the most recent summary, run:
+```
+oc logs $(oc get pods | grep ibm-eamhub-operator | awk '{print $1}') --tail 5000 | grep -A 1 '^PLAY RECAP' | tail -n 1
+```
+{: codeblock}
+
+The following indicates a loop that completed with no operations taking place (In its current state, the **PLAY RECAP** will always show **changed=1**):
+```
+$ oc logs $(oc get pods | grep ibm-eamhub-operator | awk '{print $1}') --tail 5000 | grep -A 1 '^PLAY RECAP' | tail -n 1
+localhost                  : ok=51   changed=1    unreachable=0    failed=0    skipped=11   rescued=0    ignored=0
+```
+{: codeblock}
+
+Review these 3 fields when making configuration changes:
+* **changed**: When greater than **1** indicates the operator performed a task that altered the state of one or more resources (this could be at your request by altering the custom resource, or the operator reverting a manual change that was made).
+* **rescued**: A task failed, however it was a known possible failure and the task will be attempted again on the next loop.
+* **failed**: On initial install there are some expected failures, if you are repeatedly seeing the same failure and the message is not clear (or hidden) this likely indicates a problem.
+
+### EamHub Common Configuration Options
```

Comment:
I know it is a nit, but headers should not contain all init cap words, for example, this should be: EamHub common configuration options. This is a global comment.